models/staging/stg_github_contributions.yml

```diff
@@ -7,11 +7,10 @@ models:
         tests:
           - not_null
           - unique
-      - name: user_login
-# TODO(anyone) temporarily disabled test because it keeps failing, debug and revert this later
-#        tests:
-#          - accepted_values:
-#              values: "{{ var('author_string').split(',') }}"
+      - name: lower(user_login)
+        tests:
+          - accepted_values:
+              values: "{{ var('author_string').split(',') | map(\"lower\") | list }}"
```

Comment:
does the following also exists: "{{ var('author_string').lower().split(',') }}"?